### PR TITLE
GEODE-4709: Add list gateways output sample in UG

### PR DESCRIPTION
**Pull request by alb3rtobr**
`list gateways` command had no output example in the documentation. This PR also fixes other formatting errors in the `list.html.md.erb` file

### CODE REVIEW

**Review by davebarnes97 — CHANGES_REQUESTED** [review]
Thank you for your contribution. I downloaded the changes and generated the user guide.
Thank you for improving the table and example formats in several places.
I would like to request a few improvements. Some of these issues were not introduced in your work, but I discovered them while reviewing so include them here:
- `list clients` Delete redundant intro line.
- `list functions` Some format issues in examples
- `list gateways` Change contributor's username to generic string
-  `list indexes`  Some format issues in examples
I will send you an edit that addresses these points so you can review the specifics and adopt those of which you approve.

One other issue you may wish to address in this PR:
- Output of the `list gateways` command shows a heading "Sender's Connected".
The apostrophe is out of place and should be deleted: "Senders Connected". I understand this entails a code change.

**Comment by alb3rtobr** [comment]
>     * Output of the `list gateways` command shows a heading "Sender's Connected".
>       The apostrophe is out of place and should be deleted: "Senders Connected". I understand this entails a code change.

Good catch! I have verified that this typo was already changed in the code, but not in the user guide. Apart from your suggested changes in `list.html.md.erb`, my last commit also modifies `setting_up_a_multisite_system.html.md.erb` to update the `list gateways` command output. I also changed the contributor's name that was present in the output as you did.

File: geode-docs/topologies_and_comm/multi_site_configuration/setting_up_a_multisite_system.html.md.erb
Comment (davebarnes97) [inline geode-docs/topologies_and_comm/multi_site_configuration/setting_up_a_multisite_system.html.md.erb:338]: One format correction: please insert a blank line following "Execute the load-balance command:" so Markdown will properly recognize the first line of the example code. Thanks!

**Review by davebarnes97 — APPROVED** [review]
Approved with one format correction. No need for another review cycle if you simply insert a blank line following "Execute the load-balance command:" before checking in.

**Comment by onichols-pivotal** [comment]
@alb3rtobr fyi if you had created your PR with the "allow edits from maintainers" box checked, Dave or I could fix small things like this missing blank line and save some back-and-forth.

File: geode-docs/topologies_and_comm/multi_site_configuration/setting_up_a_multisite_system.html.md.erb
Comment (alb3rtobr) [reply geode-docs/topologies_and_comm/multi_site_configuration/setting_up_a_multisite_system.html.md.erb:339]: Done, thanks for the review!

**Comment by alb3rtobr** [comment]
> @alb3rtobr fyi if you had created your PR with the "allow edits from maintainers" box checked, Dave or I could fix small things like this missing blank line and save some back-and-forth.

@onichols-pivotal thanks for the tip, I did not know that option exists :+1:

**Pull request merged**
